bigbluebutton-html5/imports/ui/components/video-provider/component.jsx

```diff
@@ -275,23 +270,14 @@ class VideoProvider extends Component {
 
   startResponse(message) {
     const id = message.cameraId;
-    const webRtcPeer = this.webRtcPeers[id];
-
-    if (message.sdpAnswer == null || webRtcPeer == null) {
-      return log('debug', 'Null sdp answer or null webrtcpeer');
-    }
+    const peer = this.webRtcPeers[id];
 
     log('info', 'SDP answer received from server. Processing ...');
 
-    webRtcPeer.processAnswer(message.sdpAnswer, (error) => {
+    peer.processAnswer(message.sdpAnswer, (error) => {
       if (error) {
         return log('error', error);
```

Comment:
Looks like we do not process the answer. Should we?

Reply:
We do process the answer, this is some logic needed by the browser webrtc peer code. The extra callback is just some code that runs after the processing was done.